package.json

```
@@ -50,21 +50,19 @@
     "node": ">=6.0.0"
   },
   "peerDependencies": {
-    "@angular/common": "^7.0.0",
-    "@angular/core": "^7.0.0",
+    "@angular/common": "~8.0.3",
+    "@angular/core": "~8.0.3",
```

Please set versions to `^8.0.0` for all `peerDependencies`
```suggestion
    "@angular/core": "^8.0.0",
```